firebase-perf/src/test/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidatorTest.java

```diff
@@ -189,6 +189,14 @@ public void testInvalidCustomAttribute() {
     trace.clearCustomAttributes().putCustomAttributes("ga_test", "value");
     assertThat(new FirebasePerfTraceValidator(trace.build()).isValidPerfMetric()).isFalse();
 
+    trace = trace.clone();
+    trace.clearCustomAttributes().putCustomAttributes("key", "");
```

Comment:
Can we have tests for when key/values are null?

Reply:
Added!